utils/migrations.js

```diff
@@ -0,0 +1,347 @@
+String.prototype.addTab = function() {
+  return `${this.valueOf()}\t`;
```

Comment:
Change tabs to 2 spaces. The end of the file do not have a new line, it must have. Also, there is a bunch of lines without semi-colon in the end.